docs/UsersGuide/source/Tutorial.rst

```diff
@@ -0,0 +1,589 @@
+     PREDEF_GRID_NAME: SUBCONUS_Ind_3km
+     DATE_FIRST_CYCL: '2019061518'
+     DATE_LAST_CYCL: '2019061518'
+     FCST_LEN_HRS: 12
+     PREEXISTING_DIR_METHOD: rename
+     VERBOSE: true
+     COMPILER: intel
+
+.. _CronNote:
+
+.. note::
+
+   Users may also want to set ``USE_CRON_TO_RELAUNCH: true`` and add ``CRON_RELAUNCH_INTVL_MNTS: 3``. This will automate submission of workflow tasks when running the experiment. However, not all systems have :term:`cron`. 
+
+``EXPT_SUBDIR:`` This variable can be changed to any name the user wants. This tutorial uses ``control`` to establish a baseline, or "control", forecast. Users can choose any name they want, from "gfsv16_physics_fcst" to "forecast1" to "a;skdfj". However, the best names will indicate useful information about the experiment. For example, this tutorial helps users to compare the output from two different forecasts: one that uses the FV3_GFS_v16 physics suite and one that uses the FV3_RRFS_v1beta physics suite. Therefore, "gfsv16_physics_fcst" could be a good alternative directory name.
+
+``PREDEF_GRID_NAME:`` This experiment uses the SUBCONUS_Ind_3km grid, rather than the default RRFS_CONUS_25km grid. The SUBCONUS_Ind_3km grid is a high-resolution grid (with grid cell size of approximately 3km) that covers a small area of the U.S. centered over Indianapolis, IN. For more information on this grid, see :numref:`Section %s <SUBCONUS_Ind_3km>`.
+
+For a detailed understanding of other ``workflow:`` variables, see :numref:`Section %s <workflow>`.
+
+In the ``workflow_switches:`` section, turn on the plotting task by setting ``RUN_TASK_PLOT_ALLVARS`` to true. All other variables should remain as they are. 
+
+.. code-block:: console
+
+   workflow_switches:
+     RUN_TASK_MAKE_GRID: true
+     RUN_TASK_MAKE_OROG: true
+     RUN_TASK_MAKE_SFC_CLIMO: true
+     RUN_TASK_GET_OBS_CCPA: false
+     RUN_TASK_GET_OBS_MRMS: false
+     RUN_TASK_GET_OBS_NDAS: false
+     RUN_TASK_VX_GRIDSTAT: false
+     RUN_TASK_VX_POINTSTAT: false
+     RUN_TASK_VX_ENSGRID: false
+     RUN_TASK_VX_ENSPOINT: false
+     RUN_TASK_PLOT_ALLVARS: true
+
+For a detailed understanding of the ``workflow-switches:`` variables, see :numref:`Section %s <workflow-switches>`.
+
+In the ``task_get_extrn_ics:`` section, add ``USE_USER_STAGED_EXTRN_FILES`` and ``EXTRN_MDL_SOURCE_BASEDIR_ICS``. Users will need to adjust the file path to reflect the location of data on their system (see :numref:`Section %s <Data>` for locations on `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ systems). 
+
+.. code-block:: console
+
+   task_get_extrn_ics:
+     EXTRN_MDL_NAME_ICS: FV3GFS
+     FV3GFS_FILE_FMT_ICS: grib2
+     USE_USER_STAGED_EXTRN_FILES: true
+     EXTRN_MDL_SOURCE_BASEDIR_ICS: /path/to/UFS_SRW_App/develop/input_model_data/FV3GFS/grib2/${yyyymmddhh}
+
+For a detailed understanding of the ``task_get_extrn_ics:`` variables, see :numref:`Section %s <task_get_extrn_ics>`.
```

Comment:
"understanding" --> "description"?

Reply:
Feel free to resolve conversation if you don't like this suggestion :)

Reply:
Np! I changed it--just forgot to say so. ;)